mako/cmd.py

```diff
@@ -24,7 +24,7 @@ def _exit():
 
 def cmdline(argv=None):
 
-    parser = ArgumentParser("usage: %prog [FILENAME]")
+    parser = ArgumentParser("mako-render [FILENAME]")
```

Comment:
the argument to ArgumentParser should be removed entirely, it generates an appropriate string automatically.

Reply:
Updated!